inst/stan/functions/exponential_decay_corr_func.stan

```diff
@@ -0,0 +1,14 @@
+/**
+  * Assembles the correlation matrix with an exponential decay corr. func.
+  * @param dist_matrix matrix with distances.
+  * @param phi parameter for function.
+  * @param l paramter for function.
```

Comment:
```suggestion
  * @param l parameter for function.
```

Reply:
Looks like this didn't get fixed? Also, would suggest being a bit more verbose here, e.g.:

```suggestion
  * @param l power to which to which to raise dist_matrix / phi before negating and exponentiating. l = 1 yields standard exponential decay.```